Source/WebCore/platform/graphics/cocoa/CMUtilities.mm

```diff
@@ -55,24 +55,34 @@
 constexpr uint32_t kAudioFormatVorbis = 'vorb';
 #endif
 
-#if ENABLE(OPUS) || ENABLE(VORBIS)
-
-static RetainPtr<CMFormatDescriptionRef> createAudioFormatDescription(const AudioInfo& info)
+CAAudioStreamDescription audioStreamDescriptionFromAudioInfo(const AudioInfo& info)
 {
+    ASSERT(info.codecName.value != kAudioFormatLinearPCM);
```

Comment:
It is not clear to me how we prevent calling `audioStreamDescriptionFromAudioInfo` with a codec that is not PCM.
Maybe we should return an optional which would be null for PCM?